scripts/src/javatests/org/oppia/android/scripts/regex/RegexPatternValidationCheckTest.kt

```diff
@@ -2153,6 +2158,148 @@ class RegexPatternValidationCheckTest {
+        android:background="@color/color_defs_shared_primary_text_color"
+        android:background="@color/color_palette_primary_text_color"
+        app:tint="@color/component_color_shared_primary_text_color"
+        app:tint="@color/color_defs_shared_primary_text_color"
+        app:tint="@color/color_palette_primary_text_color"
+        app:strokeColor="@color/component_color_shared_primary_text_color"
+        app:strokeColor="@color/color_defs_shared_primary_text_color"
+        app:strokeColor="@color/color_palette_primary_text_color"
+        app:cardBackgroundColor="@color/component_color_shared_primary_text_color"
+        app:cardBackgroundColor="@color/color_defs_shared_primary_text_color"
+        app:cardBackgroundColor="@color/color_palette_primary_text_color"
+        android:background="@color/component_color_shared_primary_text_color"
+        android:background="@color/color_defs_shared_primary_text_color"
+        android:background="@color/color_palette_primary_text_color"
+      """.trimIndent()
+    tempFolder.newFolder("testfiles", "app", "src", "main", "res", "layout")
+    val stringFilePath = "app/src/main/res/layout/test_layout.xml"
+    tempFolder.newFile("testfiles/$stringFilePath").writeText(prohibitedContent)
+
+    val exception = assertThrows(Exception::class) {
+      runScript()
+    }
+
+    // Verify that all patterns are properly detected & prohibited.
+    assertThat(exception).hasMessageThat().contains(REGEX_CHECK_FAILED_OUTPUT_INDICATOR)
+    assertThat(outContent.toString().trim())
+      .isEqualTo(
+        """
+        $stringFilePath:2: $doesNotReferenceColorFromComponentColorInLayouts
+        $stringFilePath:3: $doesNotReferenceColorFromComponentColorInLayouts
+        $stringFilePath:5: $doesNotReferenceColorFromComponentColorInLayouts
+        $stringFilePath:6: $doesNotReferenceColorFromComponentColorInLayouts
+        $stringFilePath:8: $doesNotReferenceColorFromComponentColorInLayouts
+        $stringFilePath:9: $doesNotReferenceColorFromComponentColorInLayouts
+        $stringFilePath:11: $doesNotReferenceColorFromComponentColorInLayouts
+        $stringFilePath:12: $doesNotReferenceColorFromComponentColorInLayouts
+        $stringFilePath:14: $doesNotReferenceColorFromComponentColorInLayouts
+        $stringFilePath:15: $doesNotReferenceColorFromComponentColorInLayouts
+        $stringFilePath:17: $doesNotReferenceColorFromComponentColorInLayouts
+        $stringFilePath:18: $doesNotReferenceColorFromComponentColorInLayouts
+        $wikiReferenceNote
+        """.trimIndent()
+      )
+  }
+
+  @Test
+  fun testFileContent_kotlinFiles_includesNonColorComponentReferences_fileContentIsNotCorrect() {
+    val prohibitedContent =
+      """
+        R.color.component_color_shared_activity_status_bar_color
+        R.color.color_def_avatar_background_1
+        R.color.color_palette_primary_color
+      """.trimIndent()
+    tempFolder.newFolder(
```

Comment:
@BenHenning I used here this format as the path crossing the lint limit "Exceeded max line length (100)".
That's why I arrange in this order, Is it good or there is any other way to fix the lint?

Reply:
Nope, this looks like the correct way to format it.